### PR TITLE
Descriptives: Added bar plots for continuous vars

### PR DESCRIPTION
**Pull request by jonathon-love**
I limited it to a single 'split variable' for the time being.

jonathon

### CODE REVIEW

File: R/descriptives.b.R [hunk 1]
```diff
@@ -520,47 +517,92 @@ descriptivesClass <- R6::R6Class(
...
+                    if ( ! is.factor(column)) {
 
-                        if (length(levels) > 0) {
+                        values <- data[[var]]
 
-                            columns <- na.omit(c(var, splitBy[1:3]))
-                            groups <- data[columns]
+                        nSplits <- length(splitBy)
+                        if (nSplits > 1)  # limit to one for now
+                            nSplits <- 1
```
Comment (raviselker) [inline R/descriptives.b.R]: Change both `nSplits` to 3

File: R/descriptives.b.R [hunk 2]
```diff
@@ -754,41 +796,56 @@ descriptivesClass <- R6::R6Class(
...
-                # if (self$options$barCounts)
-                #     plot <- plot + geom_text(aes_string(label=names$y), position=position_dodge(.75), vjust=-0.3)
+                if (type == 'categorical') {
 
-            }
+                    plot <- ggplot(data=data, aes_string(x=names$x, y=names$y, fill=names$s1)) +
+                        geom_bar(stat="identity", position=pd, width = 0.7, color='#333333') +
+                        labs(x=labels$x, y='counts', fill=labels$s1)
 
-            if (length(splitBy) > 1)
```
Comment (raviselker) [inline R/descriptives.b.R:788]: Leave this in (this should work for both categorical and continuous variables)

File: R/descriptives.b.R [hunk 3]
```diff
@@ -754,41 +796,56 @@ descriptivesClass <- R6::R6Class(
             names <- image$state$names
             labels <- image$state$labels
             splitBy <- self$options$splitBy
+            type <- `if`(identical(image$state$type, 'continuous'), 'continuous', 'categorical')
 
             fill <- theme$fill[2]
             color <- theme$color[1]
-            if (length(splitBy) == 2) {
```
Comment (raviselker) [inline R/descriptives.b.R]: Leave this statement in

**Review by raviselker — CHANGES_REQUESTED** [review]
So if you change a couple of things you can easily make the bar plot facet functionality work for continuous vars

**Comment by jonathon-love** [comment]
github is clearly confused, because there should only be two commits here, but whatevs.

https://github.com/jonathon-love/jmv

**Review by raviselker — APPROVED** [review]

**Pull request merged**